src/amo/components/AddonDetail.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+
+import translate from 'core/i18n/translate';
+
+import AddonMeta from 'amo/components/AddonMeta';
+import InstallButton from 'disco/components/InstallButton';
+import LikeButton from 'amo/components/LikeButton';
+import ScreenShots from 'amo/components/ScreenShots';
+import SearchBox from 'amo/components/SearchBox';
+
+
+import 'amo/css/AddonDetail.scss';
+
+export class AddonDetail extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object,
+  }
+
+  render() {
+    const { i18n } = this.props;
+
+    return (
+      <div className="AddonDetail">
+        <header>
+          <SearchBox />
+          <div className="icon">
+            <img alt="" />
+            <LikeButton />
+          </div>
+          <div className="title">
+            <h1>Placeholder Add-on Title
+              <span className="author">by <a href="#">AwesomeAddons</a></span></h1>
+            <InstallButton slug="placeholder" />
+          </div>
+          <div className="description">
+            <p>Lorem ipsum dolor sit amet, dicat graece partiendo cu usu.
+            Vis recusabo accusamus et.</p>
+          </div>
+        </header>
+
+        <section className="addon-metadata">
+          <h2 className="visually-hidden">Extension Metadata</h2>
```

Comment:
this title will one day be localized, right? I think it would be good to wrap it in `gettext` because otherwise strings like this are hard to track down.